packages/agoric-cli/src/start.js

```diff
@@ -529,7 +529,7 @@ export default async function startMain(progname, rawArgs, powers, opts) {
     ]);
     const soloAddr = rawSoloAddr.trimRight();
     const economyProposals = JSON.parse(
-      economyTemplJson.replace(/@FIRST_SOLO_ADDRESS@/g, soloAddr),
+      economyTemplJson.replace(/@PRIMARY_ADDRESS@/g, soloAddr),
```

Comment:
it's not clear why this name should change

Reply:
Because it's now used outside agsolo (which "solo" suggests)

"First" also implies a sequence and there's no sequence when this is passed in